compiler/rustc_codegen_cranelift/src/common.rs

```diff
@@ -480,7 +480,7 @@ impl<'tcx> LayoutOfHelpers<'tcx> for RevealAllLayoutCx<'tcx> {
         if let LayoutError::SizeOverflow(_) | LayoutError::ReferencesError(_) = err {
             self.0.sess.span_fatal(span, err.to_string())
         } else {
-            span_bug!(span, "failed to get layout for `{}`: {}", ty, err)
+            self.0.sess.span_fatal(span, format!("failed to get layout for `{}`: {}", ty, err))
         }
     }
```

Comment:
is the diagnostic unhelpful if we just remove the `if` condition and always do `self.0.sess.span_fatal(span, err.to_string())`?

Reply:
That was what I did initially, but since it changed a bunch of output I got blocked by having to bless tests on platforms I can't run. Perhaps having it tackled in a separate PR by someone who's better setup to do the blessing might be a good idea?